package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "pages-redirects",
+  "version": "1.0.0",
+  "main": "build-config.js",
+  "license": "CC0-1.0",
+  "scripts": {
+    "eslint": "eslint",
+    "test": "faucet test/unit/",
+    "test-docker": "faucet test/integration/",
+    "test-prod": "TARGET_HOST=https://pages.18f.gov faucet test/integration/"
+  },
+  "dependencies": {
+    "eslint": "3.16.1",
+    "eslint-config-airbnb-base": "^11.1.1",
+    "eslint-plugin-import": "2.2.0",
+    "faucet": "^0.0.1",
```

Comment:
I don't know why anyone would ever use Windows, but `faucet` doesn't seem to work on it.  It's ok though, those folks can just run the test files directly and parse the raw TAP output.